src/components/Dropdown/Dropdown.md

```diff
@@ -31,3 +31,16 @@ By default the dropdown menu is vertically aligned with the left edge of the dro
     <a href="#">Item four</a>
 </Dropdown>
 ```
+
+### Toggle On Click Inside Modifier
+
+By default the dropdown menu will no close when an item within the dropdown is clicked. Set the 'toggleDropdownOnClickInside' option to true to specify the dropdown should be closed in this case.
+
+```jsx
+<Dropdown toggleDropdownOnClickInside={true}>
```

Comment:
Since this is a boolean property you could change this example to remove the `={true}` part.